devcards.js

```diff
@@ -5,42 +5,56 @@ var $ = React.createElement;
 
 var DevCards = require('./lib/components/DevCards');
 
-exports = module.exports = devcard;
-exports.run = run;
-
-/**
- * Cards
- */
-
-// Main card
-function devcard(name, doc, body) {
-  devcard.current.push({name: name, doc: doc, body: body});
+var DEVCARDS_DIV_ID = '__devcards-root';
+
+// Singleton state
+// TODO: eliminate?
+var catalog = {};
+
+exports.ns = ns;
+function ns(name) {
+  // Because execution of a loaded module is synchronous
+  // we can expect a reloading namespace to be filled
+  // in a single pass of the event loop
+  // therefore we can enqueue a reRender for the next tick
+  var cards = catalog[name] = [];
+  function add(card) {
+    cards.push(card);
+    enqueueRender();
+  }
+  return createDefcard(add);
 }
 
-// Noop card
-devcard.off = Function.prototype;
+function createDefcard(add) {
```

Comment:
I guess `defcard` only make sense in a Clojure environment because of the `def` `defn` etc. and the macros. But maybe keeping similar names would help people.